docs/platform/HAL/connectivity.md

```diff
@@ -1 +1,97 @@
 # Connectivity
+
+You can implement various connectivity features, such as Bluetooth, WLAN, and NFC.
+
+## Bluetooth
+
+Bluetooth is a short range communication protocol used to communicate between 2 devices. Tizen uses open source Bluetooth components, such as BlueZ and ObexD. Bluez and ObexD run as a daemon and an interface library, Bluetooth Framework, is used for applications to access them over the D-Bus interface.
+
+This section explains the Bluetooth architecture on the Tizen platform and how Tizen can be ported, along with the configuration parameters and their values.
+
+**Figure: Tizen Bluetooth architecture**
+
+![Tizen Bluetooth architecture](media/800px-bluetooth.png)
+
+The Bluetooth framework provides a dialogue for the user to control the BlueZ, ObexD, and PulseAudio daemons. Bluetooth provides a standard interface between the Bluetooth chip and AP, called the HCI (Host Controller Interface). HCI can be implemented on USB, UART, and SDIO, but for the mobile environment, UART is the most common. HCI activation can differ depending on the chip vendor. The vendor provides the HCI configuration and the initial scripts. For example, Broadcom and Spreadtrum provide firmware and a loading tool. Tizen supports Bluetooth version 5.0, and the GATT, FTP, OPP, MAP, PBAP, A2DP, AVRCP, HSP/HFP, RFCOMM, HID, HDP, and PAN profiles.
+
+The Tizen Bluetooth framework is based on the open source BlueZ project. BlueZ provides the DBUS API and based on it, Tizen Bluetooth framework provides the C Language API. Using the Tizen Bluetooth framework is recommended.
+
+The following components are necessary for Bluetooth:
+
+- Application
+  - User dialogue that controls the BlueZ, ObexD, and PulseAudio daemons
+- ObexD
+  - Open source component
+  - Object exchange daemon
+  - Supports OPP, FTP, PBAP, SYNC, and MAP profile stack
+- BluetoothD
+  - BluetoothD is the open source component, BlueZ 5.79 is supported
+  - Bluetooth central daemon
+  - Supports GAP, SDP, A2DP, AVRCP, HFP, HSP, and GATT profile stack
+- Bluetooth subsystem
+  - Provides the BT unix socket. Each protocol can be accessed by its socket.
+  - Supports the L2CAP, RFCOMM, SCO, and HCI protocols
+- Bluetooth driver
```

Comment:
How about draw the block 'Bluetooth driver' into Linux kernel in the left part. (Interfaced with HAL Backend)